src/markitdown/api.py

```diff
@@ -0,0 +1,32 @@
+from fastapi import FastAPI, File, UploadFile, HTTPException
+from fastapi.responses import FileResponse
+from markitdown import MarkItDown
+import os
+
+app = FastAPI()
+
+@app.post("/convert")
+async def convert(file: UploadFile = File(...)):
+    if not file.filename:
+        raise HTTPException(status_code=400, detail="No file uploaded")
+
+    try:
+        contents = await file.read()
+        temp_file_path = f"/tmp/{file.filename}"
+        with open(temp_file_path, "wb") as temp_file:
+            temp_file.write(contents)
+
+        markitdown = MarkItDown()
+        result = markitdown.convert(temp_file_path)
```

Comment:
when this is a long CPU-bound operation (for example when converting a long PDF) this can block the executor